tracer/build/_build/GeneratePackageVersions/PackageVersionGenerator.cs

```diff
@@ -90,18 +94,30 @@ where IsSupported(entry, version.ToString(), framework)
 
         static IEnumerable<(TargetFramework framework, IEnumerable<Version> versions)> SelectMax<T>(
             IEnumerable<(Version version, TargetFramework framework)> orderedPackageVersionsByFramework,
-            Func<Version, T> groupBy)
+            Func<Version, T> groupBy,
+            IEnumerable<Version> alwaysIncludeVersions)
         {
             return orderedPackageVersionsByFramework
                   .GroupBy(x => x.framework)
                   .Select(x =>
                    {
                        var framework = x.Key;
+                       var requiredVersions = x
+                                             .Where(v => alwaysIncludeVersions.Contains(v.version))
+                                             .Select(v => v.version);
                        var versions = x
                                      .Select(v => v.version)
                                      .GroupBy(groupBy)
                                      .Select(group => group.Max());
-                       return (framework, versions);
+
+                       IEnumerable<Version> finalOrderedVersions = requiredVersions
+                                                 .Concat(versions)
+                                                 .Distinct()
+                                                 .OrderBy(x => x.Major)
+                                                 .ThenBy(x => x.Minor)
+                                                 .ThenBy(v => v.Revision)
+                                                 .ThenBy(v => v.Build);
```

Comment:
I think we can order by the `Version` itself.
```suggestion
                                                 .OrderBy(x => x);
```

Reply:
(for a moment there I thought `x` and `v` where different things, too)

Reply:
Good point, fixed in https://github.com/DataDog/dd-trace-dotnet/pull/2311/commits/8284c5142abc5c0f1fc7ac808ac27a9135507fae (along with another case of it)